addon/index.js

```diff
@@ -48,6 +48,7 @@ const URL = require('sdk/url').URL;
 const IdeaTown = require('idea-town');
```

Comment:
On that note, if we remove this `require()`, we can remove it from the [package.json](https://github.com/mozilla/idea-town/blob/master/addon/package.json#L21) as well:
```
➜  addon git:(355-clientside-metrics) ✗ depcheck
Unused Dependencies
* idea-town
```